libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -198,6 +206,7 @@ void AP_OAPathPlanner::avoidance_thread()
         bool res = false;
         switch (_type) {
         case OA_PATHPLAN_DISABLED:
+        default:
```

Comment:
Not having the default meant that the compiler could warn you if you added a case that wasn't covered here. In general I actually like not including default when we get into large switches like this.

Reply:
fixed